.github/workflows/update-release-notes.yaml

```
@@ -37,3 +59,16 @@ jobs:
           branch-suffix: random
           title: "New release notes for ${{ github.event.client_payload.application-name }}"
           delete-branch: true
+
+      - name: Auto approve
+        if: ${{ steps.create-pull-request.outputs.pull-request-operation == 'created' }}
+        run: gh pr review --approve "${{ steps.create-pull-request.outputs.pull-request-number }}"
+        env:
+          GH_TOKEN: ${{ steps.release-notes-app-token.outputs.token }}
+      - name: Pull Request Activate Automerge
```

Berre for å vere konsekvent med ekstra linjeskift mellom kvart step.
```suggestion
      
      - name: Pull Request Activate Automerge
```